proto/desmos/profiles/v3/msg_server.proto

```diff
@@ -70,4 +74,353 @@ service Msg {
   //
   // Since: Desmos 5.0.0
   rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
-}
\ No newline at end of file
+}
+
+// --------------------------------------------------------------------------------------------------------------------
+
+// MsgSaveProfile represents a message to save a profile.
+message MsgSaveProfile {
```

Comment:
structures of `MsgServices` must be inside the same file as `MsgServices`, or `cosmos.msg.v1.signer` cannot be detected properly.